### PR TITLE
ci: remove hint/needs-author-input when the author comments

### PR DESCRIPTION
**Pull request by laurentsenta**
Contributes to https://github.com/testground/testground/issues/1511

### CODE REVIEW

File: .github/workflows/pr-update-on-author-input.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - pull_request_review_comment
+
+jobs:
+  remove_label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions-ecosystem/action-remove-labels@2ce5d41b4b6aa8503e285553f75ed56e0a40bae0 # v1.3.0
+        if: github.event.comment.user.id == github.event.issue.user.id || github.event.comment.user.id == github.event.pull_request.user.id
+        with:
+          labels: hint/needs-author-input
```
Comment (galargh) [inline .github/workflows/pr-update-on-author-input.yml:14]:
```suggestion
          labels: hint/needs-author-input

```

**Comment by laurentsenta** [comment]
Tested in: https://github.com/laurentsenta/testground/pull/3

That would update the PR labels on comments to the diff as well.

File: .github/workflows/pr-update-on-author-input.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: Update PR on author input
+
+on:
+  - issue_comment
+  - pull_request_review_comment
```
Comment (galargh) [inline .github/workflows/pr-update-on-author-input.yml:5]: Don't they double each other? As in: isn't `pull_request_review_comment` also an `issue_comment`?

**Review by galargh — APPROVED** [review]

File: .github/workflows/pr-update-on-author-input.yml (hunk 2 above)
Comment (laurentsenta) [reply .github/workflows/pr-update-on-author-input.yml:5]: nope, `pull_request_review_comment` would be triggered on comments to the diff.

**Pull request merged**